README.md

```diff
@@ -36,3 +36,33 @@ iocage fetch -P jenkins -g https://github.com/ix-plugin-hub/iocage-plugin-index
 </pre>
 where *interface* is the name of the active network interface and *IP address* is the desired IP address for the plugin.
 For example, `ip4_addr="igb0|192.168.0.91"`
+
+# Plugin installation tests
+This repository contains cirrus-ci tasks for automated plugin installation tests executed as a part of the PR checks and on the `master` branch.
+
+[.cirrus/install_script.sh](.cirrus/install_script.sh) is the installation test script run for every plugin included in the [.cirrus.yml](.cirrus.yml) file.
+
+## Plugin block
+The following `.cirrus.yml` task shows an example of needed YAML section to enable automatic installation test for a plugin.
+
+```yaml
+<PLUGIN_NAME>_task:
+  <<: *INSTALL_PLUGIN
+  only_if: "changesInclude('<PLUGIN_NAME>.json', '.cirrus/install_script.sh')"
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-2
+  env:
+    PLUGIN_FILE: "<PLUGIN_NAME>.json"
+```
+
+### Block variables
+*  `<PLUGIN_NAME>` should equal (and be replaced with) the plugin JSON file name (without the `.json` file suffix)
+* The `matrix` -> `freebsd_instance` -> `image_family` should match the plugin manifest FreeBSD version. Alternatively this `matrix` can be a list of `freebsd_instances` with different versions if multiple FreeBSD versions are supported or needed to be tested
+* **required** `env` variables:
+  * `PLUGIN_FILE`: should equal the plugin manifest file and is the one parsed inside the `install_script`
+* **optional** `env` variables:
+  * `ADMIN_UI_USER`: should be used if there is a username/password required to reach the admin UI. Value should have the format: `<username>:<password>`
+  * `FOLLOW_REDIRECTS`: if set to `"false"` will not follow redirects to admin UI and treat a 3XX code as success for the admin UI check
+  * `SKIP_UI_CHECK`: if set to `"true"` will skip the admin portal control entirely
```

Comment:
Can we please improve the text a bit, maybe saying admin portal UI check - i don't think "control" is helpful in assuming that

Reply:
Sure thing, don't know why I wrote "control" there to begin with. Should be updated to "admin portal UI check" now instead.